queries.sql

```diff
@@ -118,4 +118,94 @@ GROUP BY o.full_name
 ORDER BY total_animals DESC
 LIMIT 1;
 
+------------------------------------------------------------------------------------------------------------------------------------------------
+----------------------------------------------------------3rd DAY QUERIES BELOW!----------------------------------------------------------------
+------------------------------------------------------------------------------------------------------------------------------------------------
+
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name, ve.name, v.date_of_visit
+FROM animals a 
+JOIN visits v ON a.id = v.animal_id
+JOIN vets ve ON v.vet_id = ve.id
+WHERE ve.name = 'William Tatcher'
+ORDER BY date_of_visit DESC
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT a.name, ve.name, v.date_of_visit
+FROM animals a 
+JOIN visits v ON a.id = v.animal_id
+JOIN vets ve ON v.vet_id = ve.id
+WHERE ve.name = 'Stephanie Mendez';
```

Comment:
- [ ] This query only tells us the name of the animal, vet name and date of visit but the requirement is for how many **different** animals the vet saw. Please kindly adjust query to match requirement